src/cmd/MVP-4/main.go

```diff
@@ -46,46 +45,40 @@ func main() {
 	go vehicle.Listen(vehicleOutput)
 
 	boardMux := board.NewMux(board.WithInput(vehicleOutput), board.WithOutput(vehicle.SendOrder))
-	log.Println("New Mux")
 
 	updateChan := make(chan vehicle_models.Update)
 	go boardMux.Listen(updateChan)
-	log.Println("Mux Listen")
 
-	idToType := getIdToType(podData)
+	// Communication with front-end
+	websocketBroker := websocket_broker.Get()
 
-	connectionTransfer, connectionChannel := connection_transfer.New()
-	vehicle.OnConnectionChange(connectionTransfer.Update)
-
-	dataTransfer, dataTransferChannel := data_transfer.New(getFPS(30))
+	connectionTransfer := connection_transfer.Get()
+	dataTransfer := data_transfer.Get()
+	logger := log_handle.Get()
+	messageTransfer := message_transfer.Get()
+	orderTransfer, orderChannel := order_transfer.Get()
 
-	messageTransfer, messageChannel := message_transfer.New()
+	websocketBroker.RegisterHandle(connectionTransfer, "connection/get")
+	websocketBroker.RegisterHandle(dataTransfer)
+	websocketBroker.RegisterHandle(logger, "logger/enable")
+	websocketBroker.RegisterHandle(messageTransfer)
+	websocketBroker.RegisterHandle(orderTransfer, "order/send")
 
-	orderChannel := make(chan vehicle_models.Order, 100)
-	_, ordChannel := order_transfer.New(orderChannel)
-
-	logger, loggerChannel := log_handle.NewLogger(log_handle_models.Config{
-		DumpSize: 7000,
-		RowSize:  20,
-		BasePath: os.Getenv("LOG_PATH"),
-		Updates:  make(chan map[string]any, 10000),
-		Autosave: time.NewTicker(time.Minute),
-	})
+	vehicle.OnConnectionChange(connectionTransfer.Update)
 
-	go func(msgChannel chan models.MessageTarget) {
+	go func() {
 		for update := range updateChan {
-			logger.Update(update.Fields)
-			if idToType[update.ID] == "data" {
+			logger.Update(update)
+			if msg, err := message_transfer_models.MessageFromUpdate(update); err != nil {
```

Comment:
Make a separate function that handles detecting the type of update instead of returning error from MessageFromUpdate